tycho-plugins/repository-utils/src/test/java/org/jboss/tools/tycho/sitegenerator/GitPropertiesTest.java

```
@@ -0,0 +1,37 @@
+package org.jboss.tools.tycho.sitegenerator;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.junit.Test;
+
+public class GitPropertiesTest {
+
+    Properties properties = new Properties();
+
+	@Test
+	public void testCommitId() {
+    	try {
+			properties.load(getClass().getClassLoader().getResourceAsStream("git.properties"));
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

why not just let it throw so it counts as a failure/error ?